cmd/minikube/cmd/start.go

```diff
@@ -856,39 +855,54 @@ func saveConfig(clusterConfig cfg.Config) error {
 }
 
 func validateDriverVersion(vmDriver string) {
-	if vmDriver == constants.DriverKvm2 {
-		cmd := exec.Command("docker-machine-driver-kvm2", "version")
-		output, err := cmd.Output()
 
-		// we don't want to fail if an error was returned,
-		// libmachine has a nice message for the user if the driver isn't present
-		if err != nil {
-			console.Warning("Error checking driver version: %v", err)
-			return
-		}
+	var (
+		driverExecutable    string
+		driverDocumentation string
+	)
+
+	switch vmDriver {
+	case constants.DriverKvm2:
+		driverExecutable = "docker-machine-driver-kvm2"
```

Comment:
Please move these constants somewhere else - preferably as close to the driver as possible. I'm afraid we may  silently break this in the future, should we rename from kvm to kvm2, for instance.

Reply:
Let me know if the last change is enough, or if you prefer to actually create a constant inside the `pkg/drivers/kvm|hyperkit`.